### ISSUE TITLE
Unable To Access Preferences In The 3 Dots Options Sidebar At Lower Res Using FF

### ISSUE THREAD

**Issue by trynet** [WordPress/gutenberg#30251]
Mac OS: 10.3.6 (High Sierra)
FF : Current
WP: 5.7
Theme: TwentyTwenty-One
Plugins: None

At 1280 X 720 I can't access the lowest parts of the 3 dots options sidebar. I can't do this either via mouse movement or the down arrow key.

As you can see in Screenshot 1 the last itemZ visible is "Manage all reusable blocks. Note that there is no scrollbar to the right of this sidebar.

However, when I Zoom In three times (from the default Actual Size) I see all items in the sidebar as in ScreenShot 2.

I have seen this issue for over one year.

![bug](https://user-images.githubusercontent.com/5448478/112494845-c0222100-8d59-11eb-96b0-9530fde767e1.png)

NOTE: I also tested Chrome and this was not an issue.

**Comment by talldan** [comment]
This is tricky to reproduce. I resized my browser to around the dimensions mentioned, and then opened the menu and sometimes there was no scrollbar, but sometimes there was a scrollbar. It seems to trigger more often immediately after resizing the browser viewport.

**Comment by trynet** [comment]
Yes, that's correct and I see what you mean.  When the height of the viewport is reduced (less than the height of the full screen) the scrollbar does appear.  But, as you saw, there certainly is an issue when the height of the viewport is the same as the height of the screen at 720px.

**Comment by jasmussen** [comment]
Can confirm, but it does appear to be a Firefox _only_ issue at the moment. The issue is a bit hard to reproduce. Observe in this GIF how when I initially start the viewport big, then resize, initially I don't see the scrollbar, and then it pops in after a while:

![ff](https://user-images.githubusercontent.com/1204802/114823726-326bab80-9dc4-11eb-81db-4c2cc13b1c03.gif)

It is easy to reproduce with a small viewport, as also outlined in the conversation above:

![ff2](https://user-images.githubusercontent.com/1204802/114823769-40213100-9dc4-11eb-9277-c3fea46f6ed3.gif)

In debugging a bit, I noticed this:

<img width="380" alt="Screenshot 2021-04-15 at 08 30 20" src="https://user-images.githubusercontent.com/1204802/114824190-d48b9380-9dc4-11eb-9628-501c7803d2d8.png">

Observe how the element is shown by the inspector to be 675px tall. But if you measure on the screen, the viewport only shows about 470px of available space. 

In other words, the popover element does not get correctly resized in Firefox. Which seems like a race condition in the popover component somewhere.

**Comment by youknowriad** [comment]
I can't reproduce this anymore. It seems fixed to me.
